ecl2df/summary.py

```diff
@@ -422,6 +424,30 @@ def df(
     return dframe
 
 
+def _df2pyarrow(dframe) -> pyarrow.Table:
+    """Construct a Pyarrow table from a dataframe, conserving metadata"""
+
+    # Datatypes to use for DATE column and all the numeric columns
+    # dt_timestamp_ms = pyarrow.timestamp("ms")
+    dt_timestamp_ms = pyarrow.timestamp("ns")  # fixme!
+    dt_float32 = pyarrow.float32()
+
+    # Build table column by column in order to conserve metadata.
+
+    field_list: List[pyarrow.Field] = []
+    field_list.append(pyarrow.field("DATE", dt_timestamp_ms))
+    column_arrays = [dframe.index.to_numpy()]
+
+    for colname in dframe.columns:
+        field_metadata = {b"smry_meta": json.dumps(dframe.attrs["meta"][colname])}
```

Comment:
the `smry` part in  `smry_meta` should not be there, as it is not generic enough, and it should be known from the context that this is a smry file.

Reply:
`meta` is maybe sufficient?

Reply:
I feel that the dictionary key should be something more specific/descriptive than just `meta` to avoid confusion. 
The entire dictionary is already being stored/accessed on `pyarrow.Field`'s `metadata` property so if we just call it `meta` we wind up with: `some_field.metadata["meta"]["is_total"]` etc.

Reply:
Changed in the current commit  so that you get direct access to the dictionary when accessing the field metadata.